docs/flow/modin/engines/ray/pandas_on_ray/frame/partition.rst

```diff
@@ -0,0 +1,25 @@
+PandasOnRayFramePartition
+"""""""""""""""""""""""""
+
+The class is specific implementation of :py:class:`~modin.engines.base.frame.partition.BaseFramePartition`,
+providing the API to perform operations on a block partition, namely, pandas DataFrame, using Ray as an execution engine.
+
+In addition to wrapping a pandas DataFrame, the class also holds the following metadata:
+
+* ``length`` - length of pandas DataFrame wrapped
+* ``width`` - width of pandas DataFrame wrapped
+* ``ip`` - node IP address that holds pandas DataFrame wrapped
+
+An operation on a block partition can be performed in two modes:
+
+* asynchronously_ - via :meth:`~modin.engines.ray.pandas_on_ray.frame.PandasOnRayFramePartition.apply`
+* lazily_ - via :meth:`~modin.engines.ray.pandas_on_ray.frame.PandasOnRayFramePartition.add_to_apply_calls`
```

Comment:
Links to functions aren't working (https://modin--2993.org.readthedocs.build/en/2993/flow/modin/engines/ray/pandas_on_ray/frame/partition.html)

Reply:
fixed